Compatibility with macOS

jenspots opened the pull request:
Before this change, running 'host' on macOS would result in a "found uuidgen: no" error. To reduce duplicate code, a for-loop was implemented.

jenspots committed:
Update host

jenspots committed:
Changed error-checking to be compatible with macOS and have less duplicate code.

jenspots committed:
Merge branch 'master' of https://github.com/jenspots/terminal-sharing

ninewise commented on host:
```diff
@@ -3,11 +3,14 @@ pairinguser="pairing@mumble.ugent.be"
-dvtm -v  > /dev/null 2>&1 || { echo 'dvtm found: no' && exit 1; }
+for COMMAND in abduco socat ssh uuidgen dvtm
+do
+	if ! command -v $COMMAND &> /dev/null
```
`&>` works only in bash, not all POSIX sh's. Could you replace this with `> /dev/null 2>&1` again?

ninewise commented on host:
```diff
@@ -3,11 +3,14 @@ pairinguser="pairing@mumble.ugent.be"
+do
+	if ! command -v $COMMAND &> /dev/null
+	then
+		echo $COMMAND 'found: no'
```
Even though `$COMMAND` is a known string here, and thus safe, I'd prefer `printf '%s found: no\n' "$COMMAND"`.

ninewise reviewed (changes requested):
The macOS issue was fixed by using `uuidgen` without `-V`, but I agree using `command` and a for loop are better.

Could you contract the for into `for ...; do` (and the if) to keep the style consistent?

jenspots committed:
More consistent and correct error-checking

- Changed all files with error checking
- Changed for and if style to be more consistent
- Changed 'echo' to 'printf'
- Changed &> to be more compatible with more shells

jenspots commented:
Thank you for your feedback. I added the changes you requested. I also changed all the other files that checked for commands for sake of consistency.

ninewise reviewed (approved):
Cool, thanks.

ninewise commented:
Cool, thanks.
